### PR TITLE
Address Review Comment to define SONIC_GLOBAL_DB_CLI in gbsyncd.sh

### PR DESCRIPTION
**Pull request by abdosi**
Why I did it:

  As part of PR  https://github.com/sonic-net/sonic-buildimage/pull/11754
```
    Change was added to use variable SONIC_DB_NS_CLI for
    namespace but that will not work since ./files/scripts/syncd_common.sh
    uses SONIC_DB_CLI. So revert back to use SONIC_DB_CLI and define new
    variable for SONIC_GLOBAL_DB_CLI for global/host db cli access

   Also fixed DB_CLI not working for namespace.

```
How I verify:
Manual Verification Test Result:

```
admin@str2-xxxx-lc1-2:~$ show feature status
Feature     State            AutoRestart     SetOwner
----------  ---------------  --------------  ----------
acms        enabled          enabled
bgp         enabled          enabled
database    always_enabled   always_enabled
dhcp_relay  disabled         enabled         local
gbsyncd     enabled          enabled
lldp        enabled          enabled
macsec      disabled         enabled         local
mux         always_disabled  enabled
pmon        enabled          enabled
radv        enabled          enabled
snmp        enabled          enabled
swss        enabled          enabled
syncd       enabled          enabled
teamd       enabled          enabled
telemetry   enabled          enabled

admin@str2-xxxx-lc1-2:~$ sonic-db-cli CONFIG_DB HGETALL 'FEATURE|gbsyncd'
{'auto_restart': 'enabled', 'has_global_scope': 'False', 'has_per_asic_scope': 'True', 'has_timer': 'False', 'high_mem_alert': 'disabled', 'state': 'enabled'}
admin@str2-xxxx-lc1-2:~$ sonic-db-cli -n asic0 CONFIG_DB HGETALL 'FEATURE|gbsyncd'
{'auto_restart': 'enabled', 'has_global_scope': 'False', 'has_per_asic_scope': 'True', 'has_timer': 'False', 'high_mem_alert': 'disabled', 'state': 'enabled'}

```

### CODE REVIEW

**Comment by shyam77git** [comment]
@abdosi 
How I verify:
Manual Verification

Could you please add the steps/triggers that were done and logs to ensure this fix? and no other impact

**Review by jimmyzhai — APPROVED** [review]

**Comment by abdosi** [comment]
> 

@shyam77git Updated the PR description.

**Review by shyam77git — APPROVED** [review]

File: files/scripts/gbsyncd.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 function startplatform() {
 
-    declare -a DbCliArray=($SONIC_DB_CLI $SONIC_DB_NS_CLI)
-    for DB_CLI in  ${DbCliArray[@]}; do
+    declare -A DbCliArray=([0]=$SONIC_GLOBAL_DB_CLI [1]=$SONIC_DB_CLI)
```
Comment (anamehra) [inline files/scripts/gbsyncd.sh:7]: looks good after this change.

**Pull request merged**